src/coreComponents/python/modules/geosx_xml_tools_package/geosx_xml_tools/main.py

```diff
@@ -5,9 +5,22 @@
 import os
 import time
 from geosx_xml_tools import xml_processor
+from typing import Callable, Any, List, Union, Tuple
```

Comment:
@TotoGaz - I've tested implementing python type hints in this file.  Any thoughts?  I did catch a couple of small errors in the docstring types when I was doing this, so it is certainly a useful tool.